__tests__/pages/measureId/coverage.test.tsx

```diff
@@ -0,0 +1,74 @@
+import { render, screen, act } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import { RouterContext } from 'next/dist/shared/lib/router-context';
+import { createMockRouter } from '../../helpers/testHelpers';
+import ClauseCoveragePage from '../../../pages/[measureId]/coverage';
+
+describe('coverage page rendering', () => {
+  it('should display back button', async () => {
+    await act(async () => {
+      render(
+        <RouterContext.Provider
+          value={createMockRouter({
+            query: { measureId: 'measure-EXM130-7.3.000', clauseCoverageHTML: 'test html' }
+          })}
+        >
+          <ClauseCoveragePage />
+        </RouterContext.Provider>
+      );
+    });
+
+    const backButton = screen.getByRole('button', { name: 'Back Button' }) as HTMLButtonElement;
+    expect(backButton).toBeInTheDocument();
+  });
+
+  it('should display header with measure id of calculated measure', async () => {
+    await act(async () => {
+      render(
+        <RouterContext.Provider
+          value={createMockRouter({
+            query: { measureId: 'measure-EXM130-7.3.000', clauseCoverageHTML: 'test html' }
+          })}
+        >
+          <ClauseCoveragePage />
+        </RouterContext.Provider>
+      );
+    });
+
+    expect(screen.getByText('Clause coverage for measure bundle: measure-EXM130-7.3.000')).toBeInTheDocument();
+  });
+
+  it('should display HTML content', async () => {
+    await act(async () => {
+      render(
+        <RouterContext.Provider
+          value={createMockRouter({
+            query: { measureId: 'measure-EXM130-7.3.000', clauseCoverageHTML: 'test html' }
+          })}
+        >
+          <ClauseCoveragePage />
+        </RouterContext.Provider>
+      );
+    });
+    const textDiv = screen.getByText('test html');
+    expect(textDiv).toBeInTheDocument();
+  });
```

Comment:
True, yeah we could probably combine them into one unit test. I think I made one of these way before the other one, but now that I'm looking at them together I agree that we should combine them since they are rendering the same thing